DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -136,6 +137,47 @@ public UserDataService(IDbConnection connection)
             return user;
         }
 
+        public DelegateUserCard? GetDelegateUserCardById(int id)
+        {
+            var user = connection.Query<DelegateUserCard>(
+                @"SELECT
+                        cd.CandidateID AS Id,
+                        cd.CandidateNumber,
+                        ct.CentreName,
+                        cd.CentreID,
+                        cd.DateRegistered,
+                        ct.Active AS CentreActive,
+                        cd.EmailAddress,
+                        cd.FirstName,
+                        cd.LastName,
+                        cd.Password,
+                        cd.Approved,
+                        cd.Answer1,
+                        cd.Answer2,
+                        cd.Answer3,
+                        cd.Answer4,
+                        cd.Answer5,
+                        cd.Answer6,
+                        cd.JobGroupId,
+                        jg.JobGroupName,
+                        cd.SelfReg,
+                        cd.ExternalReg,
+                        cd.Active,
+                        (SELECT AdminID
+                         FROM AdminUsers au
+                         WHERE au.Email = cd.EmailAddress AND au.CentreID = cd.CentreID

```

Comment:
What if a user is a delegate in one centre but an admin in another?

Reply:
Okay so we do need that condition, so that in the above case, the delegate doesn't get identified as an admin. But that did make me realise we need to discount where email is empty.

Reply:
Good point. Forgot that that might be a scenario.